vehicle/accel_brake_map_calibrator/src/accel_brake_map_calibrator_node.cpp

```diff
@@ -46,10 +46,10 @@ AccelBrakeMapCalibrator::AccelBrakeMapCalibrator(const rclcpp::NodeOptions & nod
   max_accel_ = declare_parameter<double>("max_accel", 5.0);
   min_accel_ = declare_parameter<double>("min_accel", -5.0);
   pedal_to_accel_delay_ = declare_parameter<double>("pedal_to_accel_delay", 0.3);
-  max_data_count_ = declare_parameter<int>("max_data_count", 200);
+  max_data_count_ = static_cast<int>(declare_parameter<int>("max_data_count", 200));
   pedal_accel_graph_output_ = declare_parameter<bool>("pedal_accel_graph_output", false);
   progress_file_output_ = declare_parameter<bool>("progress_file_output", false);
-  precision_ = declare_parameter<int>("precision", 3);
+  precision_ = static_cast<int>(declare_parameter<int>("precision", 3));
```

Comment:
What do you intend to use static_cast?

Reply:
I added this static_cast to handle clang-tidy warning.

Reply:
How about 
```suggestion
  precision_ = static_cast<int>(declare_parameter("precision", 3));
```